page-objects/src/components/sidebar/WelcomeContent.ts

```diff
@@ -0,0 +1,40 @@
+import { By, WebElement } from "selenium-webdriver";
+import { ViewSection } from "./ViewSection";
+import { AbstractElement } from "../AbstractElement";
+
+export class WelcomeContentButton extends AbstractElement {
+  constructor(
+    private panel: WebElement,
+    public welcomeSection: WelcomeContentSection
+  ) {
+    super(panel, welcomeSection);
+  }
+
+  public async getTitle(): Promise<string> {
+    return this.panel.getText();
+  }
+
+  public async takeAction(): Promise<void> {
```

Comment:
looks easier to just call `click` instead, no need for this method :)

Reply:
Indeed. That makes me wonder whether this class is actually even required, because `takeAction()` can be replaced with `button.click()` and `getTitle()`  with `button.getText()`, leaving the `WelcomeContentButton` only with the constructor. Not sure if it is worth having a separate class for that? Maybe just a typedef instead?